ansible/roles/postgres-migration/files/V3.8.sql

```diff
@@ -0,0 +1,198 @@
+CREATE TYPE status AS ENUM ('Draft', 'Live');
+CREATE TYPE programtype AS ENUM ('public', 'private');
+CREATE TABLE program
```

Comment:
I hope this table name is correct? Or is it something like `sunbird_programs`?

Reply:
sunbird_programs is database, program is the table name